source/general/index.rst

```diff
@@ -1,80 +1,87 @@
 
-The reason we need sphinx is so you can view your changes locally before you submit your changes as a pull request.
+-Info on Contributing to :ref:`Development <devcontribute>`.
 
-Viewing your changes locally
-----------------------------
+-Info on Contributing to :ref:`Digital Media <digicontribute>`.
 
-Once you have made some changes and you are ready to see your edits locally open terminal / command prompt and run
+-Info on Contributing to the :ref:`Wiki <wikicontribute>`.
 
-.. code:: bash
+-Info on Contributing to the :ref:`Server <servercontribute>`.
 
-    .\make.bat html
+-Info on Contributing as a :ref:`PK Staff Member <staff>`.
 
-This should build a copy of the wiki into the build folder in the root directory of your repository.
-To view the wiki double click on ``build > html > index.html``
+:ref:`FAQ <faq>`
+==================
 
-Submitting your changes 
 -----------------------
+How do I fix JedCore?
+-----------------------
+JedCore is a side-plugin not maintained by ProjectKorra, if you need assistance you should contact the maintainer of JedCore on their GitHub!
 
-Now that you are happy with your changes remember to commit them using git and make sure to put a discriptive message indicating what you have done.
-If you are fixing/completing an issue please also reference the issue number in your commit message.
-
-Now you are ready to make a pull request so go ahead and create one `here <https://github.com/ProjectKorra/ProjectKorraWiki/compare>`_.
+---------------------------------------
+How do I make a custom/addon ability?
+---------------------------------------
+ProjectKorra has a neat API with this especially in mind. You can find more info on the :ref:`Creating Custom Abilities <customabil>` page.
 
-Remember to select compare across forks as shown in the screenshot below
+-----------------------------------------------------------
+ProjectKorra isn't working as it should, how do I fix it?
+-----------------------------------------------------------
+The first thing you should do is make a bug report on our forums or GitHub. A developer will see it and try and help you resolve your issue. You can also send a message on our Discord and either a dev or a knowledgeable community member will come to assist you!
 
-.. image:: /_static/images/contributing_pr_1.png
+Can't find your question? Click here for even more :ref:`questions <faq>`!
 
-Once you are ready and all happy hit *Create pull request*
+Staff
+=======
+This wiki is maintained by our wonderful digital team. If you have any inquiries feel free to reach out to them on our Discord!
 
-.. note:: TODO:// More screenshots need to be placed here.
+- More Info on our `team`_!
 
-FAQ
-===
+- More Info on becoming a :ref:`PK Staff Member <staff>`!
 
-Staff
-=====
 
 Useful Links
-============
\ No newline at end of file
+==============
+
+- `Website`_
+- `Discord`_
+- `Twitter`_
+- `GitHub`_
```

Comment:
These links are boring and not very informational. View mine to understand what I mean better.

Reply:
Its pretty self explanatory, but fine